File: Python/median_of_two_sorted_arrays.py
class Solution(object):
    def findMedianSortedArrays(self, nums1, nums2):
        """
        :type nums1: List[int]
        :type nums2: List[int]
        :rtype: float
        """
        if nums1 == [0] * len(nums1) and nums2 == [0] * len(nums2):
            return 0.00000
        nums = sorted(nums1 + nums2)
        if len(nums) % 2 != 0:
            return nums[len(nums) // 2]
        elif len(nums) % 2 == 0:
            twoNS = float(nums[len(nums)/2] + nums[len(nums)/2 - 1])
            print(twoNS / 2)
            return twoNS / 2

File: C++/MedianofTwoSortedArrays.cpp
// Leetcode problem Link: https://leetcode.com/problems/median-of-two-sorted-arrays/

class Solution {
public:
    double findMedianSortedArrays(vector<int>& nums1, vector<int>& nums2) {
        vector<int> vec(nums1);
        for(auto i:nums2)
        {
            vec.push_back(i);
        }
        sort(vec.begin(),vec.end());
        int num=vec.size();
        double median=0;
        if(num%2==0)
        {
            median=(double)(vec[(num-1)/2] + vec[((num-1)/2)+1]) / 2;
        }
        if(num%2!=0)
        {
            median=(double)vec[(num-1)/2];
        }
        return median;
        
    }
};
